kolibri/utils/options.py

```diff
@@ -367,6 +368,11 @@ def path_list(value):
             "default": 8888,
             "envvars": ("KOLIBRI_ZIP_CONTENT_PORT",),
         },
+        "DISABLE_CUSTOM_CHANNEL_NAV": {
```

Comment:
As this is a learn plugin specific option, could we add it in, as an option there, rather to the global options? 

See here: https://github.com/learningequality/kolibri/blob/release-v0.14.x/kolibri/plugins/html5_viewer/kolibri_plugin.py#L14 and here for an example of this: https://github.com/learningequality/kolibri/blob/release-v0.14.x/kolibri/plugins/html5_viewer/options.py#L45